Source/Canvas.cpp

```diff
@@ -1144,9 +1147,9 @@ void Canvas::mouseDown(MouseEvent const& e)
         }
 
         // Update selected object in sidebar when we click a object
-        if (source && source->findParentComponentOfClass<Object>()) {
+        //if (source && source->findParentComponentOfClass<Object>()) {

```

Comment:
now it updates all the time?

Reply:
Good catch, I use a different system now to check if the selection has changed. So I can revert this.